hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/table/functional/TestMarkerBasedRollbackStrategy.java

```diff
@@ -149,6 +163,65 @@ public void testMarkerBasedRollbackAppendWithLogFileMarkers(IOType testIOType) t
     assertEquals(testIOType.equals(IOType.CREATE) ? 0 : 1, rollbackRequest.getLogBlocksToBeDeleted().size());
   }
 
+  @ParameterizedTest
+  @CsvSource(value = {"APPEND,true,true", "APPEND,true,false", "APPEND,false,true", "APPEND,false,false"})
+  public void testMarkerBasedRollbackAppendWithLogFileMarkersTableVersionSix(IOType testIOType,
```

Comment:
Modify `TestMarkerBasedRollbackStrategy#testGetRollbackRequestsWithMultipleLogFilesInOneFileGroup` to account for the new logic instead of adding this test?

Reply:
Can we have a separate class for testing marker based rollback strategy on mor table with version <8?

Reply:
I have moved the test case to `testGetRollbackRequestsWithMultipleLogFilesInOneFileGroup`. After moving there are no specific v6 tests left so we do not need to move it to separate class.